lib/parser.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```

Comment:
I know this is the purpose of the PR but what if someone is using an OS version prior to BigSur on which py2 is still the default, yet he did not install py3? You think we can, do an OS version check and select `workflow.py` and `workflow3.py` at runtime? I'm assuming `workflow.py` works for py2 and `workflow3.py` works for py3. Didn't look into it.

Reply:
I believe `workflow3` is for Alfred3, not py3 (it was there before). I suppose one can point users to a previous release for a py2 version. That also will need modifications to run on newer OS.

Reply:
Sure of course. This probably needs some doc work. I'll take it from here. Thanks.